File: PYTHON PROGRAMMING/Test27.py
# 27 PRINT SUM OF ALL ODD NUMBER BETWEEN 0 TO N
def func(limit):
    sum = 0
    for i in range(0, limit + 1):
        if i % 2 != 0:
            sum += i
            print(i,end=" ")
    print()
    print(f"sum: {sum}")
func(10)
func(20)
func(30)

File: TYPESCRIPT PROGRAMMING/Test27.ts
// 27. PRINT SUM OF ALL ODD NUMBER BETWEEN 0 TO N
const func27 = (range: number): void => {
    let sum: number = 0;
    for(let i = 0; i <= range; i++) {
        if(i % 2 !== 0) {
            sum += i;
        }
    }
    console.log(`Sum: ${sum}`);
}
func27(10);
func27(50);
func27(100);